src/main/resources/interface-specification.yml

```diff
@@ -987,6 +987,12 @@ components:
           type: string
           minLength: 10
           maxLength: 250
+        eserviceId:
+          type: string
+          format: uuid
+        consumerId:
```

Comment:
`consumerId` should not be changed, because it means to re-assign a purpose to another consumer, which is not allowed